src/python-fastui/fastui/json_schema.py

```diff
@@ -259,16 +260,26 @@ def special_string_field(
             )
         elif enum := schema.get('enum'):
             enum_labels = schema.get('enum_labels', {})
-            return FormFieldSelect(
-                name=name,
-                title=title,
-                placeholder=schema.get('placeholder'),
-                required=required,
-                multiple=multiple,
-                options=[SelectOption(value=v, label=enum_labels.get(v) or as_title(v)) for v in enum],
-                initial=schema.get('default'),
-                description=schema.get('description'),
-            )
+            if schema.get('mode') == 'radio' and not multiple:
```

Comment:
maybe we should raise an error if `'mode' == 'radio` and multiple`?

Reply:
good idea - added that